.eslintrc

```diff
@@ -0,0 +1,35 @@
+env:
+  node: true
+  browser: true
+
+rules:
+  brace-style: 2
+  comma-spacing: 1
+  eol-last: 0
+  indent: [2, 2]
```

Comment:
I don't use tabs to give a defined width. I use tabs to indicate an indent, and spaces to indicate separation between objects. I always assume that anyone can have the tab size set to any value they choose, and feel you would see no difference if you set your tabs to display as two spaces wide.